log-insights/limits.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'pganalyze Log Insights Limits'
+backlink_href: /docs/log-insights
+backlink_title: 'Log Insights'
+---
+
+Log Insights has a rate limit of 1 GB of log data per day for each server. Limits for
+each server in your organization are tracked separately.
+
+Log Insights limits are not present in the [Enterprise version](/enterprise-postgres-monitoring).
+
+
+## How Log Insights rate limiting works
+
+As the collector sends Log Insights data to pganalyze for processing, the service tracks
+the total number of bytes received for each server. Each server starts with a 1 GB quota
+that is continuously refilled at the rate of 1 GB per day.
+
+Effectively, each server has a 1 GB allotment of log data over a rolling 1 day period. This
+allows for sending bursts of log data as long as the average rate is under 1 GB per day.
+
+When incoming log data exceeds the limit for a particular server, the affected log lines
+are not recorded by pganalyze and will not show up in Log Insights. Query samples present
```

Comment:
Maybe "EXPLAIN plans and query samples" (in the UI we have these as separate tabs)